relay-server/src/actors/spooler/mod.rs

```diff
@@ -209,8 +211,12 @@ impl InMemory {
             count += current_count;
             self.used_memory -= current_size;
         }
+        self.envelope_count -= count as isize;
         relay_statsd::metric!(
-            histogram(RelayHistograms::BufferEnvelopesMemory) = self.used_memory as f64
+            histogram(RelayHistograms::BufferEnvelopesMemoryBytes) = self.used_memory as f64
+        );
+        relay_statsd::metric!(
+            gauge(RelayGauges::BufferEnvelopesMemoryCount) = self.envelope_count as u64
```

Comment:
We're casting an `isize` as `u64`. I think this panics on negative numbers. Should we use something like `u64::try_from`, and log an error if we couldn't parse it?

Reply:
" as " wraps around (does not panic), but I changed the type to `usize` (see other comment).